### ISSUE TITLE
Implement Age 1 cards for Unseen Expansion 

### ISSUE THREAD

**Issue by ultimatefiend** [micahstairs/bga-innovation#1074]
This issue will track the implementation of all Age 1 cards for the Unseen expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

- [x] 480 : Espionage
- [x] 481 : Palmistry
- [x] 482 : Proverb
- [x] 483 : Assassination
- [x] 484 : Maze
BLOCKED : #1098
- [x] 485 : Pilgrimage
- [x] 486 : Dance
- [x] 487 : Rumor
- [x] 488 : Silk
- [x] 489 : Handshake
- [x] 490 : Tomb
- [x] 491 : Woodworking
- [x] 492 : Myth
- [x] 493 : Polytheism
BLOCKED : #1098
- [x] 494 : Symbology
